install/deploy_final_non-compute_node.md

```diff
@@ -154,43 +154,22 @@ The steps in this section load hand-off data before a later procedure reboots th
             /var/log/conman \
             /var/log/zypper.log 2>/dev/null |
          sed 's_^/__' |
-         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$(date +%Y-%m-%d_%H-%M-%S).tgz"
+         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$PITBackupDateTime.tgz"
```

Comment:
```suggestion
         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-${PITBackupDateTime}.tgz"
```

Reply:
This change was made, marking as resolved.

Reply:
This change is not showing up as being made when I refresh the PR. Marking unresolved.